gobblin-runtime/src/main/java/org/apache/gobblin/scheduler/JobScheduler.java

```diff
@@ -398,7 +398,8 @@ public void scheduleJob(Properties jobProps, JobListener jobListener, Map<String
       // Schedule the Quartz job with a trigger built from the job configuration
       Trigger trigger = createTriggerForJob(job.getKey(), jobProps);
       this.scheduler.getScheduler().scheduleJob(job, trigger);
-      LOG.info(String.format("Scheduled job %s. Next run: %s.", job.getKey(), trigger.getNextFireTime()));
+      LOG.info("Scheduler trigger validation: [flowName: {} flowGroup: {}] - nextTriggerTime: {} - "
+          + "Job scheduled", job.getKey().getName(), job.getKey().getGroup(), trigger.getNextFireTime());
```

Comment:
nit: it says "trigger validation", but did it?  seems more like tracing.

Reply:
I don't think it's guaranteed that a job here has a flowName and flowGroup since this is the generic jobScheduler class. Also, the jobName and jobGroup is not necessarily the flowName and flowGroup either.

Reply:
Good pt, there should be two different log lines printed for  `JobScheduler` and `GobblinServiceJobScheduler`. Both contain job name and group, but only the latter will have the flow name and flow group. I created a util that will be overridden by the `GobblinServiceJobScheduler` class to extract the flow information for the tracing log line.